js/components/graph/Bool.js

```diff
@@ -6,19 +6,17 @@ import PropTypes from "prop-types"
 export default class Bool extends React.Component {
   /**
    * Create a boolean node.
-   * @param {string} state - The status of the node which can be active, inactive, or missing
    */
   constructor(props) {
```

Comment:
Now that you've removed the only state, you can delete this method entirely (no need to override the constructor).